.circleci/README.md

```diff
@@ -48,11 +46,10 @@ How things are setup for Chipyard
 The steps for CI to run are as follows.
 1st, build the toolchains in parallel (note: `esp-tools` is currently not used in the run).
 The docker image sets up the `PATH` and `RISCV` variable so that `riscv-tools` is the default (currently the `env.sh` script that is created at tool build is unused).
-2nd, install verilator using the `*.mk` to cache unique versions of verilator (mainly for if verilator is bumped).
-3rd, create the simulator binary.
+2rd, create the simulator binary.
 This requires the `riscv-tools` for `fesvr` and `verilator` to be able to build the binary.
 This stores all collateral for the tests (srcs, generated-srcs, sim binary, etc) to run "out of the gate" in the next job (make needs everything or else it will run again).
-4th, finally run the tests that were wanted.
+3th, finally run the tests that were wanted.
```

Comment:
```suggestion
3rd, finally run the tests that were wanted.
```

Reply:
If you're changing this anyway, `desired tests` sounds better than `tests that were wanted`